admin/server/projects.go

```diff
@@ -280,32 +295,31 @@ func (s *Server) UpdateProject(ctx context.Context, req *adminv1.UpdateProjectRe
 		return nil, status.Error(codes.PermissionDenied, "does not have permission to delete project")
 	}
 
-	// If changing the Github URL, check github app is installed and caller has access on the repo
-	if safeStr(proj.GithubURL) != req.GithubUrl {
-		_, err = s.getAndCheckGithubInstallationID(ctx, req.GithubUrl, claims.OwnerID())
-		if err != nil {
-			return nil, err
-		}
-	}
-
 	githubURL := proj.GithubURL
-	if req.GithubUrl != "" {
-		githubURL = &req.GithubUrl
+	if req.GithubUrl != nil {
+		// If changing the Github URL, check github app is installed and caller has access on the repo
+		if safeStr(proj.GithubURL) != *req.GithubUrl {
+			_, err = s.getAndCheckGithubInstallationID(ctx, *req.GithubUrl, claims.OwnerID())
```

Comment:
Actually, this func doesn't have a check that `claims.OwnerID()` is a user ID. We should add that